index.d.ts

```diff
@@ -1,8 +1,10 @@
-// Type definitions for react-country-region-selector 1.4.5 by <https://github.com/country-regions/react-country-region-selector>
+// Type definitions for react-country-region-selector 1.4.7 by <https://github.com/country-regions/react-country-region-selector>
 // Project: https://github.com/country-regions/react-country-region-selector
 // Definitions by: Kyle Davis <https://github.com/kyledavisdev>
 
-import * as React from 'react';
+import * as React from "react";
+
+type ValueType = "full" | "short";
```

Comment:
Should we export this too? Might be handy for consumers.

Reply:
Sure, I'll export it.